yogstation/code/modules/mob/living/carbon/human/species_types/preternis/preternis.dm

```diff
@@ -273,3 +274,25 @@ adjust_charge - take a positive or negative value to adjust the charge level
 		H.visible_message(span_danger("[P] deflects off of [H]!"), span_userdanger("[P] deflects off of you!"))
 		return 1
 	return 0
+
+/datum/species/preternis/get_species_description()
+	return "TODO: This is preternis description"
+
+/datum/species/preternis/get_species_lore()
+	return list(
+		"TODO: This is preternis lore"
+	)
+
+/datum/species/preternis/create_pref_unique_perks()
+	var/list/to_add = list()
+
+	// TODO
```

Comment:
**probably something about** 
self surgery and self repair from being metal
Emp immunity
Eyes functioning to give night vision
Crippling weakness to water
Can't wear shoes, but have innate magboot abilities
purges all chems after a short time
reduced oxygen needed to breathe

Ideally try to roll some of these together to reduce bloat